modules/installation-azure-config-yaml.adoc

```diff
@@ -175,93 +224,138 @@ endif::openshift-origin[]
 endif::gov[]
 ----
 ifndef::gov[]
-<1> Required. The installation program prompts you for this value.
+<1> Required. The `baseDomain` parameter specifies the base domain of your cloud provider. The installation program prompts you for this value.
 endif::gov[]
 ifdef::gov[]
 <1> Required.
 endif::gov[]
+ifndef::azure-restricted[]
 <2> If you do not provide these parameters and values, the installation program provides the default value.
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+<2> The configuration for the machines that comprise the control plane. The `controlPlane` section is a single mapping. The first line of the `controlPlane` section must not begin with a hyphen, `-`. You can use only one control plane pool. If you do not provide these parameters and values, the installation program provides the default value.
+endif::azure-restricted[]
+ifndef::azure-restricted[]
 <3> The `controlPlane` section is a single mapping, but the `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Only one control plane pool is used.
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+<3> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
++
+[IMPORTANT]
+====
+If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger virtual machine types, such as `Standard_D8s_v3`, for your machines if you disable simultaneous multithreading.
+====
+endif::azure-restricted[]
+ifndef::azure-restricted[]
 <4> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
 +
 [IMPORTANT]
 ====
 If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger virtual machine types, such as `Standard_D8s_v3`, for your machines if you disable simultaneous multithreading.
 ====
+endif::azure-restricted[]
+ifdef::azure-restricted[]
```

Comment:
I checked other Azure installation documentations and the 4th point is included in them. You can include that in this documentation as well.